README.md

```diff
@@ -41,19 +41,11 @@ fastify.register(require('fastify-accepts-serializer'), {
 })
 
 // Per-router serializers
-const config = {
-  serializer: {
-    serializers: [
-      {
-        regex: /^application\/x-protobuf$/,
-        serializer: body => AwesomeMessage.encode(AwesomeMessage.create(body)).finish()
-      }
-    ]
-  }
-}
-
-fastify.get('/request', { config }, function (req, reply) {
-  reply.send({pippo: 'pluto'})
+fastify.get('/request', function (req, reply) {
+  reply
+    .type('application/x-protobug')
+    .serializer((body) => AwesomeMessage.encode(AwesomeMessage.create(body)).finish())
+    .send({pippo: 'pluto'})
```

Comment:
These are vastly different implementations. One is reusable across multiple routes (the original configuration based approach) and the other requires repetitions across multiple route handler implementations.

Reply:
The route level implementation is copy from test file. If you want it to be shared across other route. I can modify it and extract it to be a variables.